client-dotnet/src/Controller.Api/dto/SutInfoDto.cs

```diff
@@ -47,4 +46,17 @@ public class SutInfoDto {
      */
     public UnitsInfoDto UnitsInfoDto { get; set; }
   }
+
+  //TODO: Review possible values for this enum
+  /*
+    Note: this enum must be kept in sync with what declared in
+    org.evomaster.core.output.OutputFormat
+ */
+  public enum OutputFormat {
+    JAVA_JUNIT_5,
+    JAVA_JUNIT_4,
+    KOTLIN_JUNIT_4,
+    KOTLIN_JUNIT_5,
+    JS_JEST
+  }
```

Comment:
we need to create a new one for dotnet, well, more specifically for CSharp

Reply:
Added